src/lib/bootloader/device_path.rb

```diff
@@ -0,0 +1,47 @@
+require "yast"
+
+module Bootloader
+  # Class for device path
+  #
+  # @example device path can be defined explicitly
+  #   DevicePath.new("/devs/sda")
+  # @example definition by UUID is translated to device path
+  #   dev = DevicePath.new("UUID=\"0000-00-00\"")
+  #   dev.path -> "/dev/disk/by-uuid/0000-00-00"
+  class DevicePath
+    attr_reader :path
+
+    # Performs initialization
+    #
+    # @param dev [<String>] either a path like /dev/sda or special string for uuid or label
+    def initialize(dev)
+      @path =  if dev_by_uuid?(dev)
+        # if defined by uuid, convert it
+        File.realpath(dev.gsub(/UUID="([-a-zA-Z0-9]*)"/, '/dev/disk/by-uuid/\1'))
```

Comment:
ugh, bootloader should not read real path on system. It is storage responsibility. that gsub is fine ( even I would use simple `sub`, but calling realpath is problematic. E.g. consider autoyast config mode.